### PR TITLE
make file structure closer to Node.js'

### PR DESCRIPTION
**Pull request by aduh95**
Having the file structure as close as nodejs/node one will help keeping
the code up-to-date. We can also use package imports for the require
calls to be as close as the upstream ones as possible.

### CODE REVIEW

File: bin/test_runner.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-// https://github.com/cjihrig/node/blob/484d35402de36d8d5756b244c8e5fbb8aa4c6afd/lib/internal/main/test_runner.js
+// https://github.com/nodejs/node/blob/HEAD/lib/internal/main/test_runner.js
```
Comment (juliangruber) [inline bin/test_runner.js:2]: +1 for changing the repo, I liked keeping the sha in there so you could tell exactly which file this was ported from. What do you think?

File: bin/test_runner.js [hunk 2]
```diff
@@ -8,15 +8,18 @@ const {
   ArrayPrototypeSort,
   Promise,
   SafeSet
-} = require('../lib/primordials')
+} = require('#internal/per_context/primordials')
```
Comment (juliangruber) [inline bin/test_runner.js:11]: very nice

File: lib/internal/errors.js [hunk 3]
```diff
@@ -38,14 +38,16 @@ function getMessage (key, args, self) {
     return ReflectApply(msg, self, args)
   }
 
-  const expectedLength =
-    (StringPrototypeMatch(msg, /%[dfijoOs]/g) || []).length
+  const expectedLength = (StringPrototypeMatch(msg, /%[dfijoOs]/g) || [])
+    .length
```
Comment (juliangruber) [inline lib/internal/errors.js:42]: about the style changes, shall we move away from `standard` to match node core closer (with semicolons and all)?

File: lib/internal/test_runner/tap_stream.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
+// https://github.com/nodejs/node/blob/HEAD/lib/internal/test_runner/tap_stream.js
```
Comment (juliangruber) [inline lib/internal/test_runner/tap_stream.js:1]: some of these files exist twice now

**Review by juliangruber — CHANGES_REQUESTED** [review]

File: lib/internal/errors.js (hunk 3 above)
Comment (aduh95) [reply lib/internal/errors.js:42]: Ugh, that's my VSCode config that's messing with me, trying to run Prettier on standard files

Comment (juliangruber) [reply lib/internal/errors.js:42]: I was using prettier-standard before but realized it was unmaintained

**Review by juliangruber — APPROVED** [review]

**Pull request merged**